src/pages/index.tsx

```diff
@@ -36,9 +36,13 @@ const Home: NextPage = () => {
   const [plannedMaintenanceData, setPlannedMaintenanceData] =
     useState<PlannedMaintenanceData | null>(null);
 
-  const fetchData = async (dataFilePath: string, setData: any) => {
+  const fetchData = async (
+    dataFilePath: string,
+    signal: AbortSignal,
+    setData: any
+  ) => {
     try {
-      const response = await fetch(dataFilePath);
+      const response = await fetch(dataFilePath, { signal });
```

Comment:
It should not log to the console on line 52 when there's an `AbortError`, as this is expected behaviour, so should add a condition for that.